tracer/src/Datadog.Trace/AgentProcessManager.cs

```diff
@@ -56,29 +52,56 @@ internal enum ProcessState
             Healthy
         }
 
+        /// <summary>
+        /// Invoked by the loader
+        /// </summary>
         public static void Initialize()
         {
             try
             {
-                if (string.IsNullOrWhiteSpace(TraceAgentMetadata.ProcessPath))
-                {
-                    return;
-                }
-
                 if (DomainMetadata.Instance.ShouldAvoidAppDomain())
                 {
                     Log.Information("Skipping process manager initialization for AppDomain: {AppDomain}", DomainMetadata.Instance.AppDomainName);
                     return;
                 }
 
-                if (!Directory.Exists(TraceAgentMetadata.DirectoryPath))
+                var customTracingEnabled = EnvironmentHelpers.GetBooleanEnvironmentVariable(ConfigurationKeys.AasEnableCustomTracing, false);
+                var needsDogStatsD = EnvironmentHelpers.GetBooleanEnvironmentVariable(ConfigurationKeys.AasEnableCustomMetrics, false);
+                var automaticTraceEnabled = EnvironmentHelpers.GetBooleanEnvironmentVariable(ConfigurationKeys.TraceEnabled, true);
+
+                EnvironmentHelpers.GetBooleanEnvironmentVariable(ConfigurationKeys.AasEnableCustomTracing, false);
```

Comment:
Not using the return value from this call?

Reply:
Fixed in 18c25b94dbc2afc43c86450a638b85de37b6d58e